rust/agama-server/src/network/nm/dbus.rs

```diff
@@ -71,17 +71,31 @@ pub fn connection_to_dbus<'a>(
     result.insert("match", match_config_to_dbus(&conn.match_config));
 
     if conn.is_ethernet() {
-        let ethernet_config = HashMap::from([(
-            "assigned-mac-address",
-            Value::new(conn.mac_address.to_string()),
-        )]);
+        let ethernet_config = HashMap::from([
+            (
+                "assigned-mac-address",
+                Value::new(conn.mac_address.to_string()),
+            ),
+            ("mtu", Value::new(conn.mtu)),
+        ]);
         result.insert(ETHERNET_KEY, ethernet_config);
     }
 
     match &conn.config {
         ConnectionConfig::Wireless(wireless) => {
             connection_dbus.insert("type", WIRELESS_KEY.into());
-            let wireless_dbus = wireless_config_to_dbus(wireless, &conn.mac_address);
+            let mut wireless_dbus = wireless_config_to_dbus(wireless);
+            wireless_dbus
+                .get_mut(WIRELESS_KEY)
+                .unwrap()
```

Comment:
If possible, I would avoid unwrapping (and potentially crashing). You can use a default value or ignore it completely.

Reply:
I mean currently this is impossible, but for future-proofing, sure.